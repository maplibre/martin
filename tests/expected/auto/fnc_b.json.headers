content-encoding: br
content-type: application/json
etag: W/"ad-WumWK2dl9zyB9VKNT00Jkg=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
