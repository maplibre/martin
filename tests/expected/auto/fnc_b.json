{
  "description": "public.function_zxy_query_jsonb",
  "name": "function_zxy_query_jsonb",
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/function_zxy_query_jsonb/{z}/{x}/{y}"
  ]
}
